torchci/pages/failure.tsx

```diff
@@ -16,6 +18,20 @@ dayjs.extend(utc);
 
 const fetcher = (url: string) => fetch(url).then((res) => res.json());
 
+export function formatFailureCaptures(failureCaptures: string) {
+  // Format the failure captures to be string[] for the API
+  let captures: any = [];
+  try {
+    captures = JSON.parse(failureCaptures);
+  } catch (e) {
+    captures = failureCaptures as string;
```

Comment:
Why not just have `captures = [failureCaptures as string]` here and get rid of line 29-31?

Reply:
On the off chance that some put puts in a quoted string, the json parse returns a string and is satisfied but it has the wrong type.  I've changed the parsing slightly so a quoted string searches for that literally but I'm not sure what the intended behavior should be